$OpenBSD: patch-scripts_coqmktop_ml,v 1.1 2013/02/07 09:43:15 chrisz Exp $

Compiling with camlp4 is broken in 8.4 and 8.4pl1. The patch below is needed
for things to work. A better approach is to migrate to camlp5.

Discussion here:
https://sympa.inria.fr/sympa/arc/coq-club/2013-01/msg00067.html

Patch here:
https://gforge.inria.fr/scm/viewvc.php/branches/v8.4/scripts/coqmktop.ml?root=coq&r1=16015&r2=16122&pathrev=16122

--- scripts/coqmktop.ml.orig	Tue Dec  4 14:40:45 2012
+++ scripts/coqmktop.ml	Sat Feb  2 18:44:09 2013
@@ -45,8 +45,7 @@ let camlp4topobjs =
     [ "Camlp4Top.cmo";
       "Camlp4Parsers/Camlp4OCamlRevisedParser.cmo";
       "Camlp4Parsers/Camlp4OCamlParser.cmo";
-      "Camlp4Parsers/Camlp4GrammarParser.cmo";
-      "q_util.cmo"; "q_coqast.cmo" ]
+      "Camlp4Parsers/Camlp4GrammarParser.cmo"]
 let topobjs = camlp4topobjs
 
 let gramobjs = []
